test/pages/Inventory.page.js

```diff
@@ -3,12 +3,49 @@ const { BaseSwagLabPage } = require('./BaseSwagLab.page');
 class InventoryPage extends BaseSwagLabPage {
     url = '/inventory.html';
 
+    // eslint-disable-next-line class-methods-use-this
     get headerTitle() { return $('.title'); }
 
+    // eslint-disable-next-line class-methods-use-this
     get inventoryItems() { return $$('.inventory_item'); }
 
+    // eslint-disable-next-line class-methods-use-this
     get addItemToCartBtns() { return $$('[id^="add-to-cart"]'); }
 
+    // eslint-disable-next-line class-methods-use-this
+    get inventoryItemsNames() { return $$('.inventory_item_name'); }
+
+    // eslint-disable-next-line class-methods-use-this
+    get inventoryItemsPrices() { return $$('.inventory_item_price'); }
+
+    // eslint-disable-next-line class-methods-use-this
+    get sort() { return $('.product_sort_container'); }
+
+    // eslint-disable-next-line class-methods-use-this
+    get activeOption() { return $('.active_option'); }
+
+    // eslint-disable-next-line class-methods-use-this
+    getSortItem(value) {
+        return $(`.product_sort_container option[value="${value}"]`);
```

Comment:
here you are chaining selectors. Please think of this approach instad of using `return $$('.inventory_item_price')`. 
This `return $$('.inventory_item_price')` feels ok in case we want to know the count, but interacting in this way could be risky.

Reply:
```
getSortItem(value) {
        return $('.product_sort_container').$(`option[value="${value}"]`);
```